src/patient-registration/patient-registration.component.tsx

```diff
@@ -31,6 +32,11 @@ export interface FormValues {
   stateProvince: string;
   country: string;
   postalCode: string;
+  contactPersonGivenName: string;
```

Comment:
Look good to me. However, we should consider abstracting some of these values into smaller interfaces as this interface is too large.

Reply:
We started working on this but it seems more complicated than expected, so maybe it would make sense to make this a story?

Reply:
Yeah, I think it would be in a different story as it's changes would probably be applied to other form fields as well.